src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -694,7 +694,10 @@ public void editConstSumQuestion(int questionNum, FeedbackConstantSumQuestionDet
     }
 
     public void verifyContributionQuestionDetails(int questionNum, FeedbackContributionQuestionDetails questionDetails) {
-        assertEquals(questionDetails.isNotSureAllowed(), getAllowNotSureContributionCheckbox(questionNum).isSelected());
+        assertEquals(questionDetails.isZeroSum(), getZeroSumCheckbox(questionNum).isSelected());
+        if (!questionDetails.isZeroSum()) {
```

Comment:
Skipping the check for "not sure" if "zero-sum" is set to true means that the test may fail to catch certain invalid cases (e.g. both "not sure" and "zero-sum" are selected). Would suggest removing this check altogether so that the values of both checkboxes are always verified, and also adding an extra check to ensure that "not sure" is not selected when "zero-sum" is set to true.